weblate/utils/tests/test_search.py

```diff
@@ -209,6 +209,7 @@ def test_has(self):
         self.assert_query("has:translation", Q(state__gte=STATE_TRANSLATED))
         self.assert_query("has:shaping", Q(shaping__isnull=False))
         self.assert_query("has:label", Q(labels__isnull=False))
+        self.assert_query("has:context", Q(context__isnull=False))
```

Comment:
The test needs to be adjusted to match the recent change :-)

Reply:
Oops… forgot. Fixed.